packages/mobile/jest.config.js

```diff
@@ -0,0 +1,55 @@
+// These are required to be transformed because
+// they are ESM and jest doesn't support ESM
```

Comment:
ive heard decent things about https://vitest.dev/, i wonder if that would support esm, cause its pretty wild just doesn't. obvs not something for rn, just a long term thing